### PR TITLE
Feature/gzip source dir

### PR DESCRIPTION
**Pull request by merryman**
Experimental branch that sees if we can speed up the "slow" load by zipping source files.

### CODE REVIEW

**Comment by linusha** [comment]
We tested this and could notice no performance benefits on either of our machines. Therefore, we'll abandon this approach for now.

**Comment by linusha** [comment]
After realizing, that this needs to be tested with `fastLoad = false`, I saw a load-time decrease of around 30% on my system.

**Comment by merryman** [comment]
I can confirm a speedup of around 25% on my machine.

File: lively.freezer/src/landing-page.cp.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ class WorldAligningLandigPageUIElements extends ViewModel {
     this.view.position = pt(0, 0);
     $world._cachedWindowBounds = null;
     document.body.style.overflowY = 'hidden';
-    await this.view.whenRendered();
+    // await this.view.whenRendered();
```
Comment (linusha) [inline lively.freezer/src/landing-page.cp.js:272]:
```suggestion
```

File: lively.freezer/src/progress-indicator.cp.js [hunk 2]
```diff
@@ -33,9 +33,10 @@ class ProgressIndicatorModel extends ViewModel {
     this.ui.progressStatus.textString = 'Loading ' + packageName;
     const maxWidth = this.ui.progressBarWrapper.width;
     const goalWidth = (packagesToLoad.indexOf(packageName) + 1) / packagesToLoad.length * maxWidth;
-    this.ui.progressFill.animate({
-      width: goalWidth, duration: 500
-    });
+    // this.ui.progressFill.animate({
+    //   width: goalWidth, duration: 500
+    // });
```
Comment (linusha) [inline lively.freezer/src/progress-indicator.cp.js:38]:
```suggestion
```

File: lively.server/plugins/dav.js [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+const tar = System._nodeRequire('tar-fs');
+import stream from 'stream';
+import util from 'util';
+import zlib from 'zlib';
 
 const COMPRESSABLE_URLS = [
   'components_cache'
 ];
 
+const compression = 'gzip'; // 'gzip';
```
Comment (linusha) [inline lively.server/plugins/dav.js:14]:
```suggestion
const compression = 'gzip'; // can be either 'gzip' or 'brotli' where 'brotli' takes drastically longer to compress but yields slightly smaller bundles;
```

**Review by linusha — APPROVED** [review]

**Pull request merged**